README.md

```diff
@@ -1 +1,30 @@
-# korerorero-front-end
\ No newline at end of file
+This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/zeit/next.js/tree/canary/packages/create-next-app).
+
+## Getting Started
+
+First, run the development server:
+
```

Comment:
I tried both `npm run dev` & `yarn dev` & they both came up with errors & the server didn't start, so:

Either need to run `npm install` (I did this then the errors were fixed)
(or potentially `yarn install`? I didn't try this tho).

Reply:
Good spotting. I've updated the readme.